<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InAndOutbord</class>
 <widget class="QWidget" name="InAndOutbord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>商品入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background:rgb(133, 133, 198);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>商品编号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background:rgb(133, 133, 198);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>商品数量:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>130</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="transButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background:rgb(85, 170, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>我要出库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background:rgb(133, 133, 198);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>关于出/入库</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
